src/com/google/javascript/jscomp/js/util/reflectobject.js

```diff
@@ -20,8 +20,9 @@
  * Definition for object reflection. See goog.reflect.object.
  *
  * @param {!Function} type Type to cast to.
```

Comment:
@concavelenz has pointed out that the return type should be related to the `type` parameter not to the  `object` parameter.

Ideally I think we should have this, but I suspect it doesn't work.
```javascript
 * @param {function(new:T, ...var_args)} type
 * @param {?Object}
 * @param {?T}
```

Reply:
I missed this question. It needs to match [goog.reflect.object](https://google.github.io/closure-library/api/goog.reflect.html) which is set as `?Object`. I'll see if that works.

Reply:
@brad4d This is now fixed